server/store.go

```diff
@@ -123,6 +135,7 @@ func (s *storeInfo) getLocationID(keys []string) string {
 type StoreStatus struct {
 	*pdpb.StoreStats
 
+	Blocked           bool      `json:"blocked"`
```

Comment:
I add a store owner to show which scheduler has acquired the store, and detect conflict when a scheduler is added.